apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/site-icon/index.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useSelect } from '@wordpress/data';
+import { Icon, wordpress } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+export default function SiteIcon(): JSX.Element {
+	const currentSiteId: number = window._currentSiteId;
```

Comment:
The `number` type is not necessary, as `_currentSiteId` is already typed [here](https://github.com/Automattic/wp-calypso/blob/93741c3debcefb5a8c317ca858d1031389dda34e/apps/editing-toolkit/typings/index.d.ts)

```suggestion
	const currentSiteId = window._currentSiteId;
```

Reply:
Thanks! I also realized using `_currentSiteId` was left over from a previous version I tried and no longer needed, so I've removed it.